ci/travis/format.sh

```diff
@@ -132,6 +142,19 @@ mypy_on_each() {
     popd
 }
 
+format_java_files() {
+  # read files from pipe
+  while read -r file; do
+      printf "%s\n" "$file" >> /tmp/ray_java_files_to_format
+  done
```

Comment:
Why not simplify it with `xargs java -jar "$GOOGLE_JAVA_FORMAT_JAR" -i`?

Reply:
I's different, we pass a file list to `google-java-foramt.jar` instead of format each file separately. Lanching jvm took some time and ` google-java-foramt.jar` supports format a file list by reading a file.

Reply:
I'm not saying to start a new java process for each file. Say if two files have changed, the xargs command will invoke `java -jar ... -i file1 file2`, not `java -jar ... -i file1` and `java -jar ... -i file2` seperately.

Reply:
I tried `git ls-files -- '*.java' "${GIT_LS_EXCLUDES[@]}" | xargs java -jar google-java-format-1.7-all-deps.jar -i`. It only took about 3 seconds.

Reply:
I already tried `java -jar ... -i file1 file2`. It will only format the first file.

Reply:
fixed